packages/internals/src/sendPanic.ts

```diff
@@ -20,7 +19,20 @@ const debug = Debug('prisma:sendPanic')
 // cleanup the temporary files even when an uncaught exception occurs
 tmp.setGracefulCleanup()
 
-export async function sendPanic(error: RustPanic, cliVersion: string, engineVersion: string): Promise<number> {
+type SendPanic = {
+  error: RustPanic
+  cliVersion: string
+  enginesVersion: string
```

Comment:
I think the old name of `engineVersion` (singular, not plural) was chosen because it is the version of _one_ engine. In theory all engines in the CLI can use different versions, and this highlights that this is specifically the version from that one engine file.

Reply:
I'm glad we discussed this then! I'm creating a private issue about it